### ISSUE TITLE
How to get a custom resource object?

### ISSUE THREAD

**Issue by greyireland** [portto/aptos-go-sdk#27]
example lp resource
```
{
  "type": "0x190d44266241744264b964a37b8f09863167a12d3e70cda39376cfb4e3561e12::liquidity_pool::LiquidityPool<0xf22bede237a07e121b56d91a491eb7bcdfd1f5907926a9e58338f964a01b17fa::asset::WETH, 0x1::aptos_coin::AptosCoin, 0x190d44266241744264b964a37b8f09863167a12d3e70cda39376cfb4e3561e12::curves::Uncorrelated>",
  "data": {
    "coin_x_reserve": {
      "value": "120720489"
    },
    "coin_y_reserve": {
      "value": "2543058577240"
    },
    "dao_fee": "33",
    "fee": "30",
    "last_block_timestamp": "1667533407",
    "last_price_x_cumulative": "437929619665727922872547093989",
    "last_price_y_cumulative": "1702452483005368160082",
    "locked": false,
    "lp_burn_cap": {
      "dummy_field": false
    },
    "lp_mint_cap": {
      "dummy_field": false
    },
    "x_scale": "0",
    "y_scale": "0"
  }
}
```

**Comment by greyireland** [comment]
Data use interface{}, or allow the user to pass in the returned structure, like json.Unmarshal() ?

**Comment by greyireland** [comment]
fix at #28

**Comment by jiazhen-lin** [comment]
The PR was merged, thanks! @greyireland
